src/main/java/gov/nasa/pds/registry/common/dd/Pds2EsDataTypeMap.java

```diff
@@ -74,7 +74,7 @@ private String guessEsDataType(String pdsType)
         if(pdsType.contains("_date")) return "date";
         if(pdsType.contains("_boolean")) return "boolean";        
         
-        return "keyword";
+        throw new DataTypeNotFoundException("Could not guess type and not more defaulting to keyword (harvest#204)");
```

Comment:
When do we have this case ? Isn't the previous case where the field is not found in an LDD enough ?

Reply:
The if list is not comprehensive and that is how we run into problems with keyword. Maybe when when pdsType is blank the contains is always false.

No, otherwise we not default to keyword.